### PR TITLE
Styling2

### PR DESCRIPTION
**Pull request by gabichuelas**
#### Description
- super lame PR that shows meager results after wrestling with bootstrap for far too long. 
- buttons are now bootstrap "primary styling".. which just means blue. 
- i changed some buttons to links so we wouldn't have huge blue buttons everywhere
- i changed the log form back to @michaeljevans 's grid because I couldn't find a better solution using our current View helpers... and im done wrestling with it for now. 
- TLDR; not much has changed.

ps. Bootstrap doesn't like `<button>` tags or `button_to` helpers.

[Heres a screenshot.](https://imgur.com/a/5cSM78H)

### CODE REVIEW

**Comment by rrabinovitch** [comment]
Thanks for messing around with this. It looks like the responsiveness to window size is working a bit better so yay for that. Only other comment I have, and maybe this is nitpicky, but it seems inconsistent to have "edit med list" and "edit profile" as links and "add new med" as a button right below.

**Comment by gabichuelas** [comment]
Making new PR for this with branch `styling3`
